frontend/src/pages/projects/screens/detail/connections/ConnectedResources.tsx

```diff
@@ -9,42 +9,69 @@ import { ProjectObjectType } from '#~/concepts/design/utils';
 import ResourceLabel from '#~/pages/projects/screens/detail/connections/ResourceLabel';
 import { getDisplayNameFromK8sResource } from '#~/concepts/k8s/utils';
 import { useInferenceServicesForConnection } from '#~/pages/projects/useInferenceServicesForConnection';
+import { PersistentVolumeClaimKind } from '#~/k8sTypes';
+import { isModelStorage } from '#~/pages/projects/screens/detail/storage/utils';
+import { useIsAreaAvailable, SupportedArea } from '#~/concepts/areas/index';
 
-type Props = {
-  connection: Connection;
-};
+export type ConnectedResourcesProps =
+  | { connection: Connection; pvc?: undefined }
+  | { connection?: undefined; pvc: PersistentVolumeClaimKind };
```

Comment:
Optional, but there is a helper type that could do this called `EitherNotBoth` in `frontend/src/typeHelpers.ts`
I think it would be something like 
```suggestion
export type ConnectedResourcesProps = EitherNotBoth<{ connection: Connection }, { pvc: PersistentVolumeClaimKind }>
```